### ISSUE TITLE
i have game yesterday not added to my mmr and not appear in my games history 

### ISSUE THREAD

**Issue by mahmoudjemmy** [ValveSoftware/Dota2-Gameplay#11872]
### Description

i have game yesterday not added to my mmr and not appear in my games history and i won the game and it unfair to didn't add it to my history game i spend almost 40 mins to win this game  i need to appear it and increase my mmr

**Comment by grozzmeister** [comment]
yep, the same happened to me, was a pretty rough game tho. By the way I thought its gonna appear after several hours as usual but unfortunately nothing happened. +25 is a thing nowadays xd I am not sure how to ask for this pts back but I am sure that no one cares about one game
